test/knossos/wgl_test.clj

```diff
@@ -84,8 +86,8 @@
 
 (deftest bad-analysis-test
   (let [a (analysis
-            (cas-register 0)
-            (ct/read-history-2 "data/cas-register/bad/bad-analysis.edn"))]
+           (cas-register 0)
+           (ct/read-history-2 "data/cas-register/bad/bad-analysis.edn"))]
```

Comment:
What's up with all the whitespace changes? Typical indent is 2 characters

Reply:
Sorry, just did a `cider-format-buffer`.